docs/formats.md

```diff
@@ -0,0 +1,75 @@
+
+The following formats are available, full details below:
+
+* `terrarium` with extention `png` in web Mercator projection, 256x256 tiles
+* `normal` with extention `png` in web Mercator projection, 256x256 tiles
+* `geotiff` with extention `tif` in web Mercator projection, 512x512 tiles
+* `skadi` with extention `hgt` in unprojected latlng, 1°x1° tiles
+
+Need help displaying raster tiles in a map? We have several [examples](display-tiles.md) using Mapzen raster tiles to style in your favorite graphics library including Tangram.
+
+## GeoTIFF
+
+**GeoTIFF** format tiles are raw elevation data suitable for analytical use and are optimized to reduce transfer costs in 512x512 tile sizes but with internal 256x256 image pyramiding, in Mercator projection (EPSG:3857). See [GDAL documentation](http://www.gdal.org/frmt_gtiff.html) for more information.
+
+Allow for the larger tile size by referring to the tile coordinate of {z-1} parent tile.
+
+## Normal
+
+**Normal** format _PNG_ tiles are processed elevation data with the the red, green, and blue values corresponding to the direction the pixel “surface” is facing (its XYZ vector), in Mercator projection (EPSG:3857). The alpha channel contains **quantized elevation data** with values suitable for common hypsometric tint ranges.
+
+* `red` = x vector
+* `green` = y vector
+* `blue` = z vector
+* `alpha` = quantized elevation data
+
+High alpha channel values indicate lower elevation values (below sea level), making them more opaque. Specifically, **normal** format alpha values are counted in (floored) elevation increments. Below sea level they start at -11,000 meters (Mariana Trench) and range to -1,000 meters in 1,000 meter increments, with more detail on the coastal shelf at -100, -50, -20, -10 and -1 meters and finally 0 (intertidal zone). Values above sea level are reported in 20 meter increments to 3,000 meters, then 50 meter increments until 6,000 meters, and then 100 meter increments until 8,900 meters (Mount Everest).
+
+Encoding quantized height ranges ([src](https://github.com/tilezen/joerd/blob/master/joerd/output/normal.py#L26-L41)):
+
+```
+for i in range(0, 11):
+    table.append(-11000 + i * 1000)
+table.append(-100)
+table.append( -50)
+table.append( -20)
+table.append( -10)
+table.append(  -1)
+for i in range(0, 150):
+    table.append(20 * i)
+for i in range(0, 60):
+    table.append(3000 + 50 * i)
+for i in range(0, 29):
+    table.append(6000 + 100 * i)
+```
+
+To decode quantized height value:
+
+  `255 - bisect.bisect_left(HEIGHT_TABLE, h)`
+
+## Skadi
+
+**Skadi** format tiles are raw elevation data in unprojected latlng (EPSG:4326) 1°x1° tiles, used by the Mapzen Terrain Service. Essentially they are the SRTMGL1 format tiles but with global coverage. See [GDAL documentation](http://www.gdal.org/frmt_various.html#SRTMHGT)) for more information.
```

Comment:
Is it worth mentioning that they're compressed with `gzip`? It is implied by the `.hgt.gz` file extension, but might be worth being explicit about it.